backend/routes/user.routes.ts

```diff
@@ -0,0 +1,60 @@
+import express = require('express');
+import { User } from '../models/user.models';
+import { auth } from '../middleware/auth';
+
+export const user3Routes = express.Router();
```

Comment:
`3` seems like a magic number, advised against in CS 101. Is there a meaning behind it? That you want 3 primary routes, /register, / login, and /logout?

Reply:
Nothing. Just a mistake. I created 3 different variations of routes, controllers, etc. When I finished up the final one, I forgot to change it.